src/sqlfluff/rules/L016.py

```diff
@@ -520,8 +520,14 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
             # these long lines will likely be single line Jinja comments.
             # They will remain as unfixable.
             if this_line[-1].type == "placeholder":
-                self.logger.info("Unfixable template segment: %s", this_line[-1])
-                return LintResult(anchor=context.segment, memory=memory)
+                if (
+                    this_line[-1].block_type != "comment"  # type: ignore[attr-defined]
```

Comment:
@tunetheweb: I changed it to a more specific `ignore[attr-defined]`, which helps explain the reason for the `ignore`. Happy to add a `cast` if you think that's better.

Reply:
Nah that’s fine. Was just curious as seemed a simple string comparison.

LGTM!